Webex Connect Flows/v3.0/Template/Media Specific Workflows/README.md

```diff
@@ -0,0 +1,84 @@
+* **Live Chat Inbound Flow** - This workflow will be triggered for every inbound customer message over the configured customer chat widget, and the widget has a live chat form configured.
+* **Live Chat Inbound Flow Without Form** - This workflow will be triggered for every inbound customer message over the configured customer chat widget and the widget doesn't have a live chat form configured.
+* **SMS Inbound Flow** - This workflow will be triggered for every inbound customer sms sent over the integrated SMS phone number.
+* **Email Inbound Flow** - This workflow will be triggered for every inbound customer email to the integrated email account.
+* **Whatsapp Inbound Flow** - This workflow will be triggered for every inbound customer message over the integrated whatsapp business number.
+* **Live Chat Close Flow** - This workflow will be triggered when a livechat customer has ended the chat.
+
+Note : 
+- Queue selection from queue task node is mandatory for any channel flows before making flow live.
+- Flow variables must be updated in the setting section with the correct asset details.
+
+## Updating Flow Variables
+
+Flow Variables that need to be updated under Custom Variables for different channels: -
+- LiveChat :- appId,liveChatDomain
+
+  <img width="800" alt="Settings" src="../../images/Settings.png">
+
+- Facebook :- FBpageid,appid
+
+  <img width="800" alt="Facebook Settings" src="../../images/Facebook.png">
+  
+- Whatsapp :- WANumber,appid
+
+  <img width="800" alt="Whatsapp Settings" src="../../images/Whatsapp.png">
+
+- Email    :- bizemailid
+
+  <img width="800" alt="Email" src="../../images/Email.png">
+
+### Live Chat Inbound Flow Details
+- In the Live Chat Flow, we need to select the template. For creation of template refer to https://help.imiconnect.io/docs/livechat-template
+Sample template is shown in below image
+
+  <img width="800" alt="Template" src="../../images/Template.png">
+
+- In the Live Chat/In App messaging node we have selected the message type as form and we have to choose the template from the dropdown which has the list of all the livechat templates available in that tenant
+
+  <img width="800" alt="formdropdown" src="../../images/formdropdown.png">
+
+- In this sample we have filled the template as LiveChatTemplate(the sample one shown in above image)
+
+  <img width="800" alt="Prechatform" src="../../images/Prechatform.png">
+
+- In the Receive node transition action we need to set the variable customerName and customerEmail and this customerName and customerEmail will 
+be visible on the agent desktop.
+
+- We are getting the value for customerName and customerEmail from the livechat form. In the given sample livechat template the customername and customeremail
+field in form is 'Name' and 'Email' so we are filling it as $(n2438.inappmessaging.formFields.Name) and $(n2438.inappmessaging.formFields.Email) respectively.
```

Comment:
`$(n2438.inappmessaging.formFields.Name) and $(n2438.inappmessaging.formFields.Email)` is not rendered correctly. Need to escape the dollar sign in markdown I guess.

<img width="1074" alt="Screenshot 2023-05-09 at 1 01 15 PM" src="https://user-images.githubusercontent.com/40895295/237026017-765c6c64-16e7-4add-8bac-4d20c6917629.png">

Reply:
Done